sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/sas/BlobServiceSasSignatureValues.java

```diff
@@ -89,27 +82,48 @@
 
     /**
      * Creates an object with empty values for all fields.
+     * @deprecated Please use {@link #BlobServiceSasSignatureValues(String)},
+     * {@link #BlobServiceSasSignatureValues(OffsetDateTime, BlobSasPermission)}, or
+     * {@link #BlobServiceSasSignatureValues(OffsetDateTime, BlobContainerSasPermission)}
      */
+    @Deprecated
```

Comment:
The decision was made to deprecate so that users dont get confused by having multiple ways to create a SAS. If we did not deprecate them, users may feel like they need to set those fields when they dont need to.